File: fundamental-data-structures/string/questions/anagram/ValidAnagramTest.kt
package string.questions.anagram

fun main() {

    val case1a = "anagram"
    val case1b = "nagaram"
    println(ValidAnagram().isAnagram(case1a, case1b))


    val case2a = "rat"
    val case2b = "car"
    println(ValidAnagram().isAnagram(case2a, case2b))

    val case3a = "anagram"
    val case3b = "nagaramm"
    println(ValidAnagram().isAnagram(case3a, case3b))
}

File: fundamental-data-structures/string/questions/anagram/validanagram_test.go
package anagram

import (
	"fmt"
	"testing"
)

func TestAnagram(t *testing.T) {
	case1a := "anagram"
	case1b := "nagaram"

	fmt.Println(isAnagram(case1a, case1b))

	case2a := "rat"
	case2b := "car"

	fmt.Println(isAnagram(case2a, case2b))
}
